US-CBP/cbp-theme
# Code markup css

#### What's this PR do?
-Example implementation of the code markup component using prism.js for review by the ui dev & creative teams.

#### Where should the reviewer start?
-CodeSnippet component under `ds-components` folder
-`.babelrc.json` (config for prism.js)


#### How should this be manually tested?
- npm install dependencies in `ds-ux-guidelines`
- npm run website:build
- navigate to the layout page under "Foundations" in the style guide
- hover over code snippet to see the copy button on the top right corner. 

#### Any background context you want to provide?
- changes in `typography.js` were mostly for testing purposes. After approval of the team I will be implementing the code snippets in there.
- The prop `language` used for CodeSnippet needs to be exact. List of languages + class name format found here: https://prismjs.com/#supported-languages


#### What are the relevant tickets?
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/13155212/78295195-b7f22580-74f9-11ea-85b4-54e90ffe3b72.png)

#### Questions:
- Is there a blog post?
- Does the knowledge base need an update?
